src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1025,6 +1025,13 @@ private BoundNode VisitTupleExpression(BoundTupleExpression node)
             return null;
         }
 
+        public override BoundNode VisitTupleBinaryOperator(BoundTupleBinaryOperator node)
+        {
+            Visit(node.Left);
+            Visit(node.Right);
+            return null;
+        }
```

Comment:
Does base visitor handle this?

Reply:
`BoundTreeVisitor` only has:
```c#
        public virtual R VisitTupleBinaryOperator(BoundTupleBinaryOperator node, A arg)
        {
            return this.DefaultVisit(node, arg);
        }
```
Also, removing this code makes the tests fail.

---
In reply to: [167980837](https://github.com/dotnet/roslyn/pull/23356#discussion_r167980837) [](ancestors = 167980837)